euth/comments/autofixtures.py

```diff
@@ -0,0 +1,19 @@
+from django.contrib.contenttypes.models import ContentType
+
+from .models import Comment
+from home.models import HomePage
+from autofixture import generators, register, AutoFixture
+
+class CommentAutoFixture(AutoFixture):
+
+    homepage_contenttype = ContentType.objects.get(app_label='home', model='homepage')
+    homepage_id = HomePage.objects.first().pk
+
+    field_values = {
+        'content_type': homepage_contenttype,
+        'object_pk': homepage_id,
```

Comment:
@slomo What do you mean by "fixture values that are somehow connected"? You could use the [AutoFixture.post_process_instance() hook](http://django-autofixture.readthedocs.io/en/latest/usage.html#autofixture.base.AutoFixture.post_process_instance) to add generic foreign keys. Does this answer your question?